src/Uno.UI/UI/Xaml/UIElement.wasm.cs

```diff
@@ -27,6 +27,75 @@ public partial class UIElement : DependencyObject
 		internal const string DefaultHtmlTag = "div";
 
 		private readonly GCHandle _gcHandle;
+		private bool _isCssStyleUpdatesSuspended = false;
+
+		/// <summary>
+		/// Since there is no way to apply all set and reset updates in bulk, we create them of groups and execute
+		/// each group separately to maintain the desired order.
+		/// </summary>
+		private readonly Queue<IQueuedUpdatesGroup> _queuedUpdates = new Queue<IQueuedUpdatesGroup>();
+
+		internal interface IQueuedUpdatesGroup
+		{
+			void Execute();
+		}
+
+		internal class SetStylesQueuedUpdates : IQueuedUpdatesGroup
+		{
+			private readonly UIElement _uiElement;
+			private readonly List<(string name, string value)> _styles = new List<(string name, string value)>();
```

Comment:
Try to avoid tuples on wasm, specifically. This uses generics and it's not efficient in current builds of the runtime.

Reply:
@jeromelaban The existing methods `SetStyle` and `ResetStyle` works with tuples. I can introduce a small struct containing the name and value, but this will probably require lots of updates over the codebase for every caller to these methods. 

Alternatively, I could introduce new overloads and leave the existing ones, but that will keep the inefficiency problem in some places.

What do you think?